_includes/v3.0/v3.0.0-beta1/release_notes.md

```diff
@@ -0,0 +1,51 @@
+
+- Calico now works with Kubernetes network services proxy with IPVS/LVS. Calico enforces network policies with kube-proxy running in IPVS mode for Kubernetes clusters. Currently only workload ingress policy is supported.
+
+- After a period of deprecation, this release removes support for the `ETCD_AUTHORITY` and `ETCD_SCHEME` environment variables. Calico no longer reads these values. If you have not transitioned to `ETCD_ENDPOINTS`, you must do so as of v3.0. Refer to [Configuring `calicoctl` - etcdv3 datastore](https://docs.projectcalico.org/v3.0/reference/calicoctl/setup/etcdv3) for more information.
+
+- A new node controller for Kubernetes deployments clears data associated with deleted nodes from the Calico datastore, preventing conflicts that can lead to crash loops. Refer to [Configuring the Calico Kubernetes controllers](https://docs.projectcalico.org/v3.0/reference/kube-controllers/configuration) for more information.
+
+- `calicoctl` now allows a `0` value for ICMP entries in policy resources, enabling `ping` responses. In addition, it now rejects `255` values in the `type` field due to lack of kernel support. Refer to the reference documentation of the [network policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy#icmp) and [global network policy](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/globalnetworkpolicy#icmp) resources for more information.
+
+
+#### Limitations
+
+- **No upgrades**: this version of Calico ends support for etcd version 2. Existing customers must migrate their data to etcd version 3. The alpha and beta releases do not provide migration capabilities, nor do they support upgrades. We plan to add migration and upgrade support in the GA release.
+
+- **Integrates only with Kubernetes, OpenShift, and host endpoints**: the OpenStack, Mesos, DC/OS, rkt, and Docker orchestrators have not been tested and are not supported. (Calico still supports Docker and rkt containers.) We plan to resume support for the OpenStack, Mesos, DC/OS, and Docker orchestrators in a future release.
+
+- **BGP route reflector not supported**: large deployments that require the [BGP route reflector](https://docs.projectcalico.org/v3.0/usage/routereflector/bird-rr-config) are not supported. We plan to resume support for the BGP route reflector in a future release.
+
+- **GoBGP not supported**: Setting the `CALICO_NETWORKING_BACKEND` environment variable to `gobgp` is not supported. See [Configuring calico/node](https://docs.projectcalico.org/v3.0/reference/node/configuration) for more information. We plan to resume support for GoBPG in a future release.
+
+<!-- Once migration from etcdv2 to etcdv3 is supported, restore the following warning -->
```

Comment:
I think we can get rid of this warning entirely since the only path to v3.0 is through v2.6.4.

Reply:
I agree, I think we can and should lose this comment.

Reply:
I don't think we can - this is still required for migration for Calico KDD v2.4.x to Calico v2.5???

Reply:
hmm, maybe. So if someone running v2.4 plans to upgrade to v2.5 and they're looking at the v3.0 docs, then I guess we do need to preserve this warning ... probably for the next 2-3 major revisions. I was thinking they'd be looking at the v2.x docs rather than v3.0.

Reply:
Well, because they _must_ upgrade to v2.6.4 first, before they can upgrade to v3.0, they indeed must look at the v2.6 docs before they look at the v3.0 docs. So, yes, I think we can delete this note. Also, I think it will be better to keep the upgrade stuff all together in the upgrade page, not sure the Release Notes is the right place for this.

Reply:
Ah, gotcha. Missed that this was in the v3.0.0 specific notes. d'oh.